pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -79,6 +79,8 @@ const (
 	metricsAggregationOldCrdName    = "kubevirtmetricsaggregations.kubevirt.io"
 	nodeLabellerBundlesOldCrdName   = "kubevirtnodelabellerbundles.kubevirt.io"
 	templateValidatorsOldCrdName    = "kubevirttemplatevalidators.kubevirt.io"
+
```

Comment:
please correct me if I'm wrong, but according to what I [see](https://github.com/kubevirt/kubevirt/blob/ddce8abb40142ccc229feb77593719965e6ff431/pkg/virt-config/config-map.go#L351) - masquarade is already default in kubeivrt.

Reply:
please note that in the link you sent, it parses the values in the configMap, and it accepts `Bridge`, `Slirp` and `Masquerade` as proper values, but it will not determine the default interface, only parsing. 
The value is determined [here](https://github.com/kubevirt/kubevirt/blob/ddce8abb40142ccc229feb77593719965e6ff431/pkg/virt-config/config-map.go#L162) and it is being set [here](https://github.com/kubevirt/kubevirt/blob/ddce8abb40142ccc229feb77593719965e6ff431/pkg/virt-config/virt-config.go)

that default configuration will take place unless who ever creates the configMap states otherwise. in this case the HCO is creating the configMap, and if we will inject `masquerade` it will not use kubevirt-default-value which is bridge.

note that if someone will deploy kubevirt without HCO it will still get bridge as default.

Reply:
ack thanks!